psychopy/app/coder/coder.py

```diff
@@ -2641,7 +2641,9 @@ def paste(self, event):
         foc = self.FindFocus()
         if hasattr(foc, 'Paste'):
             foc.Paste()
-        self.currentDoc.ConvertEOLs(BaseCodeEditor.getEOL(self.prefs['newlineConvention']))
+        EOL = BaseCodeEditor.getEOL(self.prefs['newlineConvention'])
+        if EOL is not None:
+            self.currentDoc.ConvertEOLs()
```

Comment:
Does ConvertEOLs need to be told what to convert them _to_? 
```
self.currentDoc.ConvertEOLs(EOL)
```

Reply:
Thanks, that was an error. Back to testing :)